docs/services/slam/cartographer/_index.md

```diff
@@ -272,17 +338,48 @@ See [Attributes](#attributes) for details.
 }
 ```
 
-{{% /expand%}}
+{{% /tab %}}
+{{< /tabs >}}
+
+2. Start a mapping session
+
+   Navigate to the **Control** tab on your robot's page and click on the drop-down menu matching the `name` of the service you created.
+   On the cartographer panel, you can start a mapping session.
 
-#### Localize only
+   When you start a mapping session:
 
-In this mode, the cartographer-module on your robot executes the Cartographer algorithm itself locally.
+   - Viam spins up an instance of the `cartographer` module in the cloud to execute the Cartographer algorithm.
+     Because Cartographer's algorithm is CPU-intensive when creating or updating a map, in this mode the `cartographer` module on your robot acts as a stub.
```

Comment:
Same here, I don't think we need to mention that Cartographer is CPU-intensive when creating or updating a map again here, since we already mentioned it in "Mapping Modes"